app/assets/stylesheets/documents.scss

```diff
@@ -1,3 +1,18 @@
 .red {
   color: red;
+}
+
+table{
```

Comment:
This is styling ALL of our tables. Do you know if we want all of our tables to have these css styles?